src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const newState = Object.assign({}, state);
+  const arrReturn = [];
+
+  for (const i of actions) {
```

Comment:
Doesn't such name for an action confuse you?👀

```suggestion
  for (action of actions) {
```